### ISSUE TITLE
loader fails on watch only: Module build failed: Unknown word (5:1)

### ISSUE THREAD

**Issue by FreeMasen** [webpack-contrib/css-loader#623]
I have been looking over the issues here (closed and open) as well as stack overflow questions related to this error however none of these solutions have helped. 

One of the most confusing parts about this is that most of what I can find related the the error message are consistent while my problem only happens when I use the --watch flag (a normal webpack build will work with no issue).

I have been able to setup a minimal example [here](https://github.com/FreeMasen/webpack.error.watch).
<details>
<summary>full error message</summary>

The first build here you can see is successful, however the second build output comes after I have saved (not changed) the entry file.

```powershell
PS C:\webpack.error.watch> .\node_modules\.bin\webpack --watch

Webpack is watching the files…


[at-loader] Using typescript@2.4.2 from typescript and "tsconfig.json" from D:\Robert\webpack.error.watch/tsconfig.json.


[at-loader] Checking started in a separate process...

[at-loader] Ok, 1.286 sec.
Hash: bd74121678a6dd9c15eb
Version: webpack 3.5.5
Time: 6388ms
           Asset   Size  Chunks             Chunk Names
JobReview.min.js  90 kB       0  [emitted]  JobReview
   [1] ./src/app.tsx 1.42 kB {0} [built]
   [2] delegated ../node_modules/es6-object-assign/auto.js from dll-reference vendor 42 bytes {0} [built]
   [3] delegated ../node_modules/react/react.js from dll-reference vendor 42 bytes {0} [built]
   [4] delegated ../node_modules/react-dom/index.js from dll-reference vendor 42 bytes {0} [built]
   [5] delegated ../node_modules/moment/moment.js from dll-reference vendor 42 bytes {0} [built]
   [6] delegated ../node_modules/react-big-calendar/lib/index.js from dll-reference vendor 42 bytes {0} [built]
   [7] delegated ../node_modules/react-redux/es/index.js from dll-reference vendor 42 bytes {0} [built]
   [8] ./lib/css/sos-big-calendar.css 1.05 kB {0} [built]
   [9] ./node_modules/css-loader!./lib/css/sos-big-calendar.css 14.2 kB {0} [built]
    + 4 hidden modules

[at-loader] Checking started in a separate process...

[at-loader] Checking finished with 1 errors
Hash: d724b8ed3cfcd0dd092d
Version: webpack 3.5.5
Time: 1289ms
 1 asset
   [1] ./src/app.tsx 1.42 kB {0} [built]
   [2] delegated ../node_modules/es6-object-assign/auto.js from dll-reference vendor 42 bytes {0} [built]
   [3] delegated ../node_modules/react/react.js from dll-reference vendor 42 bytes {0} [built]
   [4] delegated ../node_modules/react-dom/index.js from dll-reference vendor 42 bytes {0} [built]
   [5] delegated ../node_modules/moment/moment.js from dll-reference vendor 42 bytes {0} [built]
   [6] delegated ../node_modules/react-big-calendar/lib/index.js from dll-reference vendor 42 bytes {0} [built]
   [7] delegated ../node_modules/react-redux/es/index.js from dll-reference vendor 42 bytes {0} [built]
  [13] ./node_modules/style-loader!./node_modules/css-loader!./lib/css/sos-big-calendar.css 1.29 kB [built] [prefetched]
  [14] ./node_modules/css-loader!./lib/css/sos-big-calendar.css 340 bytes [built] [prefetched] [failed] [1 error]
  [15] ./node_modules/awesome-typescript-loader/dist/entry.js!./src/app.tsx 1.44 kB [built] [prefetched]
    + 6 hidden modules

ERROR in ./node_modules/css-loader!./lib/css/sos-big-calendar.css
Module build failed: Unknown word (5:1)

  3 | // load the styles
  4 | var content = require("!!../../node_modules/css-loader/index.js!./sos-big-calendar.css");
> 5 | if(typeof content === 'string') content = [[module.id, content, '']];
    | ^
  6 | // Prepare cssTransformation
  7 | var transform;
  8 |


ERROR in [at-loader] ./src/app.tsx:3:32
    TS2551: Property 'setPrototypeOf' does not exist on type 'ObjectConstructor'. Did you mean 'getPrototypeOf'?
```
</details>

**Comment by mkMoSs** [comment]
I have same issue #617, not limited to watch only though. I made a minimal repo that replicates the issue: https://github.com/mkMoSs/css-loader-bug

This is a major problem since it doesn't allow me to use libraries like react-toolbox, that depends on css-loader for my project.

**Comment by evilebottnawi** [comment]
@mkMoSs you have invalid configuration, just remove
```
.webpackConfig({
       module: {
           rules: [
               {
                   test: /\.css$/,
                   use: [
                       {
                           loader: "style-loader",
                       },
                       {
                           loader: "css-loader",
                       },
                   ],
               },
           ],
       },
   });
```
laravel mix already know how handle css

**Comment by evilebottnawi** [comment]
@FreeMasen `module.loaders` => `rules`, `loaders` inside `rules` => `use`. Also don't reproducible with you example. Seems you provide not full repository or use other configuration. Feel free to feedback.

**Comment by evilebottnawi** [comment]
@FreeMasen also please upgrade `style-loader` to `latest` version, `wepback` also

**Comment by FreeMasen** [comment]
Thanks for the update @evilebottnawi , I am still seeing this error though. I have updated the example repo I sent in before with a readme.md detailing the build steps to reproduce.

[repo](https://github.com/FreeMasen/webpack.error.watch)

I have updated all of the dependencies you requested and updated the package.json, maybe I am missing something?

![error output](https://user-images.githubusercontent.com/6232026/36397213-4b21dc62-1587-11e8-9186-363e14842c73.gif)


build command

```sh
$ npm i
$ webpack --config ./webpack.config.dll.js
$ webpack --config ./webpack.config.js -w
```

Save the file `app.tsx` (no change is required, just saving the fill will trigger a rebuild from webpack), notice the first build happens with no error.
